ThreeBotPackages/tfgrid/threebot_provisioning/chatflows/threebot_reservation.py

```diff
@@ -0,0 +1,135 @@
+from Jumpscale import j
+import netaddr
+import random
+import nacl
+import os
+
+
+def chat(bot):
+
+    """
+    This chat is create 3bot container using zosv2
+    """
+    explorer = j.clients.threebot.explorer
+    cl = j.clients.s3.get("deployer")
+
+    AWS_ID = cl.accesskey_
+    AWS_SECRET = cl.secretkey_
+
+    user_info = bot.user_info()
+    name = user_info["username"]
+    email = user_info["email"]
+    ips = ["IPV6", "IPV4"]
+    choose = ["New", "Restore"]
+    ip_range_choose = ["Specify IP Range", "Choose IP Range for me"]
+
+    folder_name = name.replace(".", "_")
+    env = dict()
+    secret_env = dict()
+
+    if not name or not email:
+        bot.md_show("Username or email not found in session. Please log in properly")
+
+    user_choice = bot.single_choice("create a new 3bot or restore existing 3bot?", choose)
+    identity = explorer.actors_all.phonebook.get(name=name, email=email)
+    identity_pubkey = identity.pubkey
+    if user_choice == "Restore":
+        password = bot.secret_ask("please add restore password?")
+        hash_restore = nacl.hash.blake2b(password.encode(), key=identity_pubkey.encode()).decode()
+
+    # ask user about corex user:password and ssh-key to give him full access to his container
+    pub_key = bot.string_ask(
+        "Please add your public ssh-key (that will allow you to access the deployed container using ssh) "
```

Comment:
let's help people here even more.


"Please add your public ssh key. This will allow you to access the deployed container using ssh.

Just copy your key from `~/.ssh/id_rsa.pub`